bin/smash

```diff
@@ -204,17 +205,18 @@ def main():
          if not args.dry_run:
             run_multiprocessing_engine(config,os.path.abspath(t))
 
-      if args.all_testsets or args.testset is not None:
-         smashbox.no_engine.testsets=[None]
-         execfile(os.path.abspath(t),smashbox.no_engine.__dict__)
+      for j in range(1,args.loop+1):
+         if args.all_testsets or args.testset is not None:
```

Comment:
I agree. It's an interesting information to show for debugging purposes, and easy to get with the log